test_util/aws.py

```diff
@@ -24,6 +24,13 @@ def template_by_instance_type(instance_type):
         return VPC_TEMPLATE_URL
 
 
+def param_dict_to_aws_format(user_parameters):
+    final_parameters = []
+    for k, v in user_parameters.items():
+        final_parameters.append({'ParameterKey': k, 'ParameterValue': str(v)})
+    return final_parameters
+
```

Comment:
in single line using comprehensions? 

`return [{"ParameterKey": k, "ParameterValue: str(v)} for k, v in user_parameters.items()]`